scripts/download_blenderkit.py

```diff
@@ -10,68 +10,74 @@
 from pathlib import Path
 import uuid
 
```

Comment:
This whole block should put inside of a fct. call and then use a `if __name__ == "__main__":` block to call the fct.